src/views/acquirer/legal-power.njk

```diff
@@ -0,0 +1,73 @@
+{% extends "page.njk" %}
+{%- from "govuk/components/textarea/macro.njk" import govukTextarea -%}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-l">Request ID: {{requestId}}</span>
+      <form method="POST" id="legalPowerForm">
+        {% set legalPowerDecision %}
+            {{ govukTextarea({
+                id: "legal-power-textarea",
+                name: "legal-power-textarea",
+                label: {
+                    text: "What legal power will you use?"
+                },
+                value: savedValue['legal-power'].explanation
+            }) }}
+        {% endset -%}
+        {{ govukRadios({
+            name: "legal-power",
+            fieldset: {
+                legend: {
+                    text: "Do you have the legal power to request this data?",
+                    isPageHeading: true,
+                    classes: "govuk-fieldset__legend--l"
+                }
+            },
+            hint: {
+                text: "For example, through common law or royal prerogative."
+            },
+            items: 
+            [
+            {
+                value: "yes",
+                text: "Yes, we believe we have the legal power to do this",
+                checked: savedValue['legal-power'].decision === 'yes',
```

Comment:
The `savedValue` that gets passed into the template looks like this:
`value: { decision: 'yes', explanation: 'something' }`
So I think this needs to be changed to:
`checked: savedValue.decision === 'yes',`

Reply:
Good spot! Thank you, I'll push the changes in a commit shortly 😊